packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -117,6 +127,7 @@ const FUTURE_FLAGS_DEFAULTS: { [key: string]: boolean } = {
   [DOCKER_IGNORE_SUPPORT]: false,
   [SECRETS_MANAGER_PARSE_OWNED_SECRET_NAME]: false,
   [KMS_DEFAULT_KEY_POLICIES]: false,
+  [S3_BUCKET_GRANT_WRITE_ACL]: false,
```

Comment:
Shouldn't the new project default and the V2 default be the same? We want this to become the standard behavior, yeah?

Reply:
I thought `FUTURE_FLAGS` is the one that will be used for V2? And this is only used for the defaults?

Their JSDocs certainly seem to indicate that...

Reply:
Yes but shouldn't the value be `true` then?

Reply:
If you put it as `true` here, you will change the behavior to have `s3:PutObject` instead of `s3:PutObject*` if the feature flag is not explicitly set, kind of defeating the point of having the feature flag at all... (we risk breaking people that created their CDK project before this change was out, when they update to the newest version)